src/Config/SettingsTab.ts

```diff
@@ -291,6 +291,68 @@ export class SettingsTab extends PluginSettingTab {
                     await this.plugin.saveSettings();
                 });
             });
+
+        // ---------------------------------------------------------------------------
+        containerEl.createEl('h4', { text: 'Reminder Settings' });
+        // ---------------------------------------------------------------------------
```

Comment:
I think a more logical (and more visible) position for this would be after the Date Settings section.